mmdet/models/utils/activation.py

```diff
@@ -0,0 +1,33 @@
+import torch.nn as nn
+
+activation_cfg = {
+    # format: layer_type: (abbreviation, module)
+    'relu': nn.ReLU,
+    'leaky_relu': nn.LeakyReLU,
```

Comment:
May also add PReLU, RReLU, ReLU6, SELU, CELU, GELU.

Reply:
`kaiming_init` and `inplace` option only support `relu` and `leaky_relu`.

Reply:
Yes, so we need to handle such cases in `init_weights`. Here we list all possibilities for potential usage (not necessarily in mmdet).

Reply:
All added except for GELU, which is not supported in pytorch 1.1